web/modules/custom/rdf_entity/src/Entity/Rdf.php

```diff
@@ -248,15 +248,8 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
    * {@inheritdoc}
    */
   public static function bundleFieldDefinitions(EntityTypeInterface $entity_type, $bundle, array $base_field_definitions) {
-    // @todo This is the key to proper bundle fields..!
-    // @todo Only invoked after cache clear.
-    $fields = array();
-    // If ($bundle == 'admssw_softwareproject') {.
-    $fields['test'] = BaseFieldDefinition::create('string')
-      ->setLabel(t('Test bundle admssw_softwareproject'))
-      ->setDescription(t('The ID of the Contact entity.'))
-      ->setReadOnly(TRUE);
-    // }.
+    // Allow other modules to override the label field.
+    $fields['label'] = clone $base_field_definitions['label'];
```

Comment:
Is it really necessary to do this in the RDF Entity module? Why would the standalone contributed RDF Entity module suddenly be interested in providing the label as a bundle field? What about the `rid` and the `id`? Are other modules not allowed to override these? If all are bundle fields, why define them as base fields in the first place?

It would be better to look if it is possible to do this with a hook such as `hook_entity_bundle_field_info_alter()`.